medmodels/medrecord/_overview.py

```diff
@@ -1,109 +1,150 @@
-            attribute_info = ["-"]
-        elif schema and attribute in schema:
+            attribute_info = {"values": "-"}
+        elif schema and attribute in schema and schema[attribute][1]:
             if schema[attribute][1] == AttributeType.Continuous:
-                attribute_info = [
-                    f"min: {attribute_values.min()}",
-                    f"max: {attribute_values.max()}",
-                    f"mean: {attribute_values.mean():.{decimal}f}",
-                ]
+                attribute_info = _extract_numeric_attribute_info(attribute_values)
             elif schema[attribute][1] == AttributeType.Temporal:
-                time_attribute = attribute_values.str.to_datetime()
-                attribute_info = [
-                    f"min: {min(time_attribute).strftime('%Y-%m-%d %H:%M:%S')}",
-                    f"max: {max(time_attribute).strftime('%Y-%m-%d %H:%M:%S')}",
-                ]
-            elif schema[attribute][1] == AttributeType.Categorical:
-                attribute_info = [
-                    _extract_string_attribute_info(
-                        attribute_series=attribute_values,
-                        long_string_suffix="categories",
-                        short_string_prefix="Categories",
-                    )
-                ]
+                attribute_info = _extract_temporal_attribute_info(attribute_values)
             else:
-                attribute_info = [
-                    _extract_string_attribute_info(attribute_series=attribute_values)
-                ]
-
-        ## Without Schema
+                attribute_info = _extract_string_attribute_info(
+                    attribute_series=attribute_values,
+                    long_string_suffix="categories",
+                    short_string_prefix="Categories",
+                )
+        # Without Schema
         else:
             if attribute_values.dtype.is_numeric():
-                attribute_info = [
-                    f"min: {attribute_values.min()}",
-                    f"max: {attribute_values.max()}",
-                    f"mean: {attribute_values.mean():.{decimal}f}",
-                ]
+                attribute_info = _extract_numeric_attribute_info(attribute_values)
             elif attribute_values.dtype.is_temporal():
-                attribute_info = [
-                    f"min: {min(attribute_values).strftime('%Y-%m-%d %H:%M:%S')}",
-                    f"max: {max(attribute_values).strftime('%Y-%m-%d %H:%M:%S')}",
-                ]
+                attribute_info = _extract_temporal_attribute_info(attribute_values)
             else:
-                attribute_info = [
-                    _extract_string_attribute_info(attribute_series=attribute_values)
-                ]
+                attribute_info = _extract_string_attribute_info(
+                    attribute_series=attribute_values
+                )
+
+        data_dict[str(attribute)] = attribute_info
```

Comment:
```suggestion
        data_dict[attribute] = attribute_info
```

in the case that there is a attribute "0" and 0 for example